data_augmentation.py

```diff
@@ -11,6 +12,7 @@
 from imgaug.augmentables.kps import Keypoint, KeypointsOnImage
 
 from constants import ImageAugmentationStrength
+from constants import RL_FLIP
```

Comment:
Nit,merge the two lines

Reply:
tip: you can right click in VSCode and there's actually a "sort imports" button that merges it for you

Reply:
Didn't know this existed ty!